common/dynamicconfig/constants.go

```diff
@@ -1260,6 +1260,24 @@ these log lines can be noisy, we want to be able to turn on and sample selective
 		100,
 		`MatchingMaxTaskQueuesInDeployment represents the maximum number of task-queues that can be registered in a single worker deployment version`,
 	)
+	MatchingPollerScalingBacklogAgeScaleUp = NewTaskQueueDurationSetting(
+		"matching.pollerScalingMinimumBacklog",
+		200*time.Millisecond,
+		`MatchingPollerScalingBacklogAgeScaleUp is the minimum backlog age that must be accumulated before 
+a decision to scale up the number of pollers will be issued`,
+	)
+	MatchingPollerScalingSyncMatchWaitTime = NewTaskQueueDurationSetting(
```

Comment:
is it still specific to sync match, or can we shorten/simplify the name?
```suggestion
	MatchingPollerScalingPollWaitTime = NewTaskQueueDurationSetting(
```